src/argus_htmx/templates/htmx_base.html

```diff
@@ -6,6 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block title %}<title>Argus Server: {{ page_title }}</title>{% endblock %}
 
+    <script src="{% static "htmx.min.js" %}"></script>
+    <!-- HTMX Config -->
+    <meta name="htmx-config" content='{"useTemplateFragments":"true"}'>
```

Comment:
Do we need to use this? It doesn't seem like we use `<template>` tags anywhere

Reply:
It's copied from the original though so nothing changes. I'm curious myself, what is this for?